ansible/roles/cyhy_reporter/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+# tasks file for cyhy_reporter
+
+#
+# Replace commander.conf password
+#
+- name: Grab mongo users from S3
+  local_action:
+    module: aws_s3
+    bucket: ncats-cyhy-secrets
+    object: mongo_users.yml
+    dest: /tmp/mongo_users.yml
+    mode: get
+  become: no
+
+- name: Load mongo users token from YML file
+  local_action:
+    module: include_vars
+    file: /tmp/mongo_users.yml
+    name: mongo_users
+  become: no
+
+#
+# Set up /etc/cyhy/cyhy.conf
+#
+- name: Create the /etc/cyhy directory
+  file:
+    path: /etc/cyhy
+    state: directory
+
+- name: Create /etc/cyhy/cyhy.conf
+  copy:
+    dest: /etc/cyhy/cyhy.conf
+    owner: cyhy
+    group: cyhy
+    mode: 0660
+    content: |
+      [DEFAULT]
+      default-section = production
+      database-uri = mongodb://localhost:27017/
+      report-key =
```

Comment:
We need to add another S3 secret for the default report-key.  Let me know if you need any clarification on this.